### PR TITLE
[#938] Buttons, buttons everywhere

### PR DESCRIPTION
**Pull request by lucapette**
Screenshot 2021-02-12 at 14 17 58

### CODE REVIEW

File: docs/src/components/ButtonBoxList/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import styles from './styles.module.css';
+
+const ButtonBoxList = ({children}) => {
+  return (<ul className={styles.buttonBoxList}>
+  {React.Children.map(children, c  => <li className={styles.buttonBoxListItem}>{c}</li>)}
+  </ul>);
```
Comment (lucapette) [inline docs/src/components/ButtonBoxList/index.js:8]: @pascal-airy this shiny new component was brought to you with love by me and @AitorAlgorta 🙏

**Review by sgarroum — DISMISSED** [review]
perfection

**Review by sgarroum — APPROVED** [review]
true perfection

**Review by pascal-airy — APPROVED** [review]
image

**Pull request merged**